portfolio/src/main/webapp/script.js

```diff
@@ -48,25 +48,24 @@ function bubbleFall() {
     }
 }
 
-// Get messages from the data servlet.
-function getMessages() {
-  fetch('/data').then(response => response.json()).then((messages) => {
-    console.log(messages);
+// Get comments from the data servlet.
+function getComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    console.log(comments);
 
-    // Find container.
-    const messageContainer = document.getElementById('message-container');
-
-    // Add each message to container as list element.
-    for (let i = 0; i < 3; i++) {
-      messageContainer.appendChild(createListElement(messages[i]));
+    const commentSectionContainer = document.getElementById('comment-section-container');
+    
+    // Add each comment to container as comment element.
+    for(const comment of comments) {
+      commentSectionContainer.appendChild(createCommentElement(comment));
     }
   });
-}
+}S
 
-// Creates an <li> element containing text.
-// COPIED FROM SERVER-STATS EXAMPLE.
-function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
+// Creates an <comment> element containing text.
+function createCommentElement(text) {
```

Comment:
From https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments: In methods and named functions, parameter and return types must be documented